0xProject/0x-monorepo
# [instant] Base heap integration

## Description

Introduces base heap integration with instant.  Starts off with reporting on "Widget - Opened" and "Wallet - Ready", and sending in some details about network, where it was embedded from, and what the provider is.

There are some TODOs which will get resolved when we introduce the Rollbar branch which allows us to know if we are on a dev environment or not.  I made an Asana task for these.

## Testing instructions

- Check out this branch
- Log into Heap, chose "Instant - Development" project
- View live action tracking ( https://heapanalytics.com/app/live )
- Look for "Widget - Opened" event
- Watch for "Wallet - Ready" event (should only trigger on initial render and on changing wallet -- should not trigger w/ every heart beat)


## Types of changes









## Checklist:
